source/sass/views/article.scss

```diff
@@ -50,7 +50,7 @@
 .article-summary-link {
   display: block;
   --spacing: 10px;
-
+  width: 100%;
```

Comment:
Since the links had a width of "auto" shorter length links had the possibility of sharing a line with another link.

With the width now set to 100%, each link is taking up its own line.